### ISSUE TITLE
[Feature Request]: Short description of the feature

### ISSUE THREAD

**Issue by Nidhogblo** [prasanthrangan/hyprdots#1371]
## Feature Request
Asus Drivers incorporations
### Description

Hi, I work in a drivers incorporation for Asus laptops, I made a little script to detect Asus computers and the incorporation of the drivers in the computer and add this in the pacman for the updates, all work really well, but I don't know where can I put this script to incorporate in the code (I see the globals_fn.sh,but i don't know where is the correct form to push mi changes ,for do the pr)
### Use Case
This is really useful for the control of specials things of laptops asus,like a fan controler, special keys ,lights and other features 

### Proposed Solution
the connection to specials drivers to do this and the add this on pacman :)

### Additional Information
These features can be added to aspector visuals or other tools such as way bar synchronisation with fan control,etc.

the packages are:
asusctl
[asusctl](https://aur.archlinux.org/packages/asusctl/)AUR is a CLI utility for ASUS ROG & TUF laptop, to name some of the important features it gives users control over:
Integrated GPU MUX Control
Keyboard RGB Lighting Profile (but limited compared to the Windows AURA/Armoury Crate)
Fan Curves
Battery Charge Limit
Panel Overdrive
AniMe Matrix Screens

For usage instructions see [asusctl](https://wiki.archlinux.org/title/Asusctl).
supergfxctl

[supergfxctl](https://aur.archlinux.org/packages/supergfxctl/)AUR is a CLI utility for managing GPU switching functionality on ASUS hybrid laptops, particularly dedicated GPU MUX control.

For usage instructions see [supergfxctl](https://wiki.archlinux.org/title/Supergfxctl).
rog-control-center

[rog-control-center](https://aur.archlinux.org/packages/rog-control-center/)AUR is a GUI frontend for asusctl and supergfxctl.

more info:
https://wiki.archlinux.org/title/ASUS_Linux
https://asus-linux.org/guides/arch-guide

**Comment by dieBakterie** [comment]
You know there are Keys that needs to be added and stuff like that?

**Comment by dieBakterie** [comment]
You know there are Keys that needs to be added and stuff like that?

**Comment by Nidhogblo** [comment]
what?

**Comment by dieBakterie** [comment]
Like this you can do it yourself for now. Propably will be added when enough people want it to be added.
[HyprV4 by SolDoesTech](https://github.com/SolDoesTech/HyprV4/blob/main/set-hypr#L349-L373)

**Comment by kRHYME7** [comment]
I remove it for a while in the CLI just to adjust on some changes (Hyprdots > > > Hyde )

https://github.com/kRHYME7/Hyde-cli/blob/2db3ec5b5bc260d0254d5af44744961d89ad5b97/Hyde#L504C1-L521C9

**Comment by Nidhogblo** [comment]
> I remove it for a while in the CLI just to adjust on some changes (Hyprdots > > > Hyde )
> 
> https://github.com/kRHYME7/Hyde-cli/blob/2db3ec5b5bc260d0254d5af44744961d89ad5b97/Hyde#L504C1-L521C9

Yeah,I see the change and the project is really interesting ,if they need help,i could help us with code or with the documentation :D

**Comment by Nidhogblo** [comment]
> > where can I put this script to incorporate in the code (I see the globals_fn.sh,but i don't know where is the correct form to push mi changes ,for do the pr)
> 
> Hey @Nidhogblo , the packages you have listed looks very specific to some asus laptops. So if you have a quick and easy way to detect the laptop model, then you can add your logic before this line to append `install_pkg.lst` -
> 
> https://github.com/prasanthrangan/hyprdots/blob/e3a07cc30f02cd7af64c64c082ca41f2766b24c2/Scripts/install.sh#L137-L141

Hi ,I do the implementation in the code and it works fine,do you want to do a pr? ,I think my code could be better organised with the project format but for the moment it is functional and fine.

**Comment by kRHYME7** [comment]
@Nidhogblo  Hello🌊 

Can you check if ` Hyde inject asus_patch ` works. Update the cli first ` yay -Sy hyde-cli-git ` .

**Comment by abenezerw** [comment]
Can you confirm if the above solution fixes this and close this PR please. @Nidhogblo
